JOSS/refs.bib

```diff
@@ -1,83 +1,61 @@
-@article{wang2021assessing,
```

Comment:
wang2021assessing and ren2015beyond were duplicates.

All other changes are either auto-formatting or unicode correction.